### PR TITLE
Upgrade to Django4

### PR DESCRIPTION
**Pull request by gracieflores**
The changes are similar to `django-citeIt`. You will see two additional changes to models.py. One was to add a space to the return per flake8 and the other was changing the unicode method to str. This has been tested locally with `Python 3.9` using the admin to add instances of an event.

### CODE REVIEW

**Comment by gracieflores** [comment]
@ldko @somexpert This is ready for review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@ Change Log
 ==========
 
 
+1.1.0
+-----
+* Upgraded to Django 4.1
+* Removed support for Django 2.2
+* Removed support for Python 3.6 and 3.7
+* Added support for Python 3.10
```
Comment (ldko) [inline CHANGELOG.md]: For consistency within this file we could add periods to the ends of these lines.

File: major_event_log/models.py [hunk 2]
```diff
@@ -32,13 +32,13 @@ class Event(models.Model):
     contact_email = models.EmailField()
 
     def get_absolute_url(self):
-        return(reverse('major-event-log:event_details', args=[self.id]))
+        return (reverse('major-event-log:event_details', args=[self.id]))
```
Comment (ldko) [inline major_event_log/models.py]: Do we need the parentheses around the call to `reverse` or could those go away?

**Review by ldko — COMMENTED** [review]
I didn't think of it on the django-citeit upgrade, but you should add your name to the Contributors lists on these projects when you are upgrading. It is a nice contribution :D.

File: major_event_log/models.py (hunk 2 above)
Comment (gracieflores) [reply major_event_log/models.py]: We don't need them. I will remove them.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,14 @@ Change Log
 ==========
 
 
+1.1.0
+-----
+* Upgraded to Django 4.1.
+* Removed support for Django 2.2.
+* Removed support for Python 3.6 and 3.7.
+* Added support for Python 3.10.
```
Comment (somexpert) [inline CHANGELOG.md:10]: Maybe we should say we're adding support for 3.8 through 3.10?

**Review by somexpert — COMMENTED** [review]
I think the changes look good, but I think we also need to adjust the docs directory so that [this](https://django-major-event-log.readthedocs.io/en/latest/index.html) stays up to date. Also had one minor comment about the change log.

File: CHANGELOG.md (hunk 3 above)
Comment (gracieflores) [reply CHANGELOG.md:10]: I think that is mentioned below, but it says `Added Python 3.8 and 3.9 to testing.`. Should I still add it to the new notes?

Comment (somexpert) [reply CHANGELOG.md:10]: oh, yep, you're right. I think it's fine the way it is then

**Review by ldko — APPROVED** [review]
Works for me. Thanks @gracieflores :+1:

**Review by somexpert — APPROVED** [review]
Looks good to me! :thumbsup:

**Pull request merged**